shared-doc/attributes.adoc

```diff
@@ -28,7 +28,9 @@ ifdef::ProductRelease[]
 :productNameFull: Red Hat JBoss Enterprise Application Platform
 :productVersion: 7.4
 :DocInfoProductNumber: {productVersion}
-
+:githubRepoUrl: https://github.com/jboss-developer/jboss-eap-quickstarts
```

Comment:
this is defined already a few lines below in the IF product release or xp release, and thus overrides this definition

Reply:
fixed